docarray/array/storage/annlite/backend.py

```diff
@@ -134,3 +134,6 @@ def __setstate__(self, state):
 
     def __len__(self):
         return self._annlite.index_size
+
+    def close(self):
+        self._annlite.close()
```

Comment:
I do not think there is a `close` interface in DocArray

Reply:
Yes, this is only for annlite backend. We need this because the file lock needed to be closed before access it again.